python/tests/xir/test_integration.py

```diff
@@ -14,10 +14,10 @@ def parse_script(circuit: str) -> XIRProgram:
 
 
 photonics_script = """
-gate Sgate, 2, 1;
-gate BSgate, 2, 2;
-gate Rgate, 1, 1;
-output MeasureHomodyne;
+gate Sgate(a, b)[0];
+gate BSgate(theta, phi)[0, 1];
+gate Rgate(p0)[0];
```

Comment:
Also, do we need a test where one of the wires is a string? (apologies if I missed it)